running 'make check' tries to delete my system libraries

No shit, it did.

tar -xf enchant-2.2.1.tar.gz
cd enchant-2.2.1
./configure --prefix=/usr --libdir=/usr/lib64 --enable-relocatable
make
make check
...
rm: cannot remove '/usr/lib64/ts/dejitter.so': Permission denied
rm: cannot remove '/usr/lib64/ts/touchkit.so': Permission denied
rm: cannot remove '/usr/lib64/ts/touchkit.la': Permission denied
rm: cannot remove '/usr/lib64/ts/iir.so': Permission denied
rm: cannot remove '/usr/lib64/ts/pthres.so': Permission denied
rm: cannot remove '/usr/lib64/ts/iir.la': Permission denied
rm: cannot remove '/usr/lib64/ts/linear.la': Permission denied
rm: cannot remove '/usr/lib64/ts/skip.so': Permission denied
rm: cannot remove '/usr/lib64/ts/skip.la': Permission denied
rm: cannot remove '/usr/lib64/ts/debounce.so': Permission denied
rm: cannot remove '/usr/lib64/ts/invert.so': Permission denied
rm: cannot remove '/usr/lib64/libQt5Sql.so.5': Permission denied
rm: cannot remove '/usr/lib64/libcupsimage.so.2': Permission denied
make[4]: *** [Makefile:1550: libenchant-copy] Error 1
make[4]: Leaving directory '/building/inkscape/enchant-2.2.1/tests/enchant_providers'
make[3]: *** [Makefile:1547: usr/lib64/libenchant-2.la] Error 2
make[3]: Leaving directory '/building/inkscape/enchant-2.2.1/tests/enchant_providers'
make[2]: *** [Makefile:1411: check-am] Error 2
make[2]: Leaving directory '/building/inkscape/enchant-2.2.1/tests/enchant_providers'
make[1]: *** [Makefile:1833: check-recursive] Error 1
make[1]: Leaving directory '/building/inkscape/enchant-2.2.1/tests'
make: *** [Makefile:865: check-recursive] Error 1

--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -33,7 +33,7 @@ $(LIBENCHANT_COPY): $(top_builddir)/src/libenchant-@ENCHANT_MAJOR_VERSION@.la
 	$(MAKE) libenchant-copy
 
 libenchant-copy:
-	rm -rf $(libdir_subdir)/
+	rm -rf $(libdir_subdir)
 	$(MKDIR_P) $(libdir_subdir)/enchant-@ENCHANT_MAJOR_VERSION@
 	cp -r $(top_builddir)/src/@objdir@ $(libdir_subdir)/
 	cp $(top_builddir)/src/libenchant-@ENCHANT_MAJOR_VERSION@.la $(libdir_subdir)/

--- a/tests/enchant_providers/Makefile.am
+++ b/tests/enchant_providers/Makefile.am
@@ -28,7 +28,7 @@ $(LIBENCHANT_COPY): $(top_builddir)/src/libenchant-@ENCHANT_MAJOR_VERSION@.la
 	$(MAKE) libenchant-copy
 
 libenchant-copy:
-	rm -rf $(libdir_subdir)/
+	rm -rf $(libdir_subdir)
 	$(MKDIR_P) $(libdir_subdir)/enchant-@ENCHANT_MAJOR_VERSION@
 	cp -r $(top_builddir)/src/@objdir@ $(libdir_subdir)/
 	cp $(top_builddir)/src/libenchant-@ENCHANT_MAJOR_VERSION@.la $(libdir_subdir)/

